AGENTS.md

```
@@ -1,76 +1,74 @@
-- **Clerk** for authentication
-- **Nodemailer** (SMTP) for server‑side email sending
-- **Progressive Web App (PWA)** features (manifest, service worker, offline support)
+- **Tailwind CSS** pour le style
+- **Clerk** pour l'authentification
+- **Nodemailer** (SMTP) pour l'envoi d'e-mails côté serveur
+- Des fonctionnalités **Progressive Web App (PWA)** (manifest, service worker, support hors-ligne)
 
-The source lives under the `src/` folder and follows a conventional Next.js layout:
+Le code source se trouve dans le dossier `src/` et suit la structure conventionnelle de Next.js :
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)